scylla/tests/integration/tablets.rs

```diff
@@ -365,7 +366,7 @@ async fn test_tablet_feedback_not_sent_for_unprepared_queries() {
 
             // I expect Scylla to not send feedback for unprepared queries,
             // as such queries cannot be token-aware anyway
-            send_unprepared_query_everywhere(
+            execute_unprepared_statement_everywhere(
                 &session,
                 session.get_cluster_data().as_ref(),
                 &Query::new(format!("INSERT INTO {ks}.t (a, b, c) VALUES (1, 1, 'abc')")),
```

Comment:
🤔 I don't really like "execute" used with unprepared statements, because EXECUTE is a CQL command to execute prepared statements. Would "send_unprepared_statement_everywhere" be an acceptable name?

Reply:
I've been thinking that once we do the execution API refactor for 2.0, there will be only one method for all kinds of statements: `execute`. Do you agree? If so, then why not use it here?

Reply:
I've been thinking about it, and I think that even in new execution API we should keep separate methods for unprepared statements, prepared statements, and batches.
Why? The choice of the query type should be more concious, a single method makes this less obvious.
Also the interface would be simpler to learn: user would still have methods of the struct that accept simple types, and not traits which then user needs to research and learn what actually implements them.

What I think request execution refactor should be mostly about is enabling configuration of the request, meaning we can do `session.execute(something).with_timestamp(....).paging_iter().await` instead of having an exponential amount of methods.

Reply:
OK, makes sense.

Reply:
OTOH, the fact that the CQL protocol has some specific names for execution of prepared and unprepared statements **does not** imply that those names are a good fit for the names of high-level user-facing API functions. It's not intuitive at all for anyone not well-versed in the CQL protocol that "query" is related to unprepared statements and "execute" to prepared statements.

If in the future we were to have distinct names for execution of different types of statements, then I'd go for:
- `execute_unprepared()`,
- `execute_prepared()`,
- `execute_batch()`.

The names then make it perfectly clear that the action is _execution_ and the object is a particular kind of a statement.

Reply:
I agree that the names do not need to be the same (but in that case the documentation should clearly state which CQL command it corresponds to).
The proposed names are a bit too long for my taste.